spec/ysh-user-feedback.test.sh

```diff
@@ -16,7 +16,7 @@ get_opt a 'lol'
 
 echo 'should not get here'
 
-## status: 1
+## status: 2
```

Comment:
Hm I wonder why these changed too

Reply:
Because `'lol'` doesn't start with the `:` sigil, it fails with a usage error. We actually get a pretty nice error message here too!

```sh
$ bin/ysh -c 'proc myproc (out Ref) { setref out = 42 }; myproc '\''lol'\'   
  proc myproc (out Ref) { setref out = 42 }; myproc 'lol'
                                             ^~~~~~
[ -c flag ]:1: fatal: Ref param 'out' expected arg starting with colon : but got 'lol'
```

But I guess this actually goes against the spirit of this test. Should we change that line to `get_opt 'lol' :a`?

Reply:
Oh yeah probably!  not sure what happened there, maybe a typo